src/node_crypto.cc

```diff
@@ -5071,6 +5071,16 @@ void InitCryptoOnce() {
   CRYPTO_set_locking_callback(crypto_lock_cb);
   CRYPTO_THREADID_set_callback(crypto_threadid_cb);
 
+#ifdef OPENSSL_FIPS
+  if (!FIPS_mode_set(1)) {
+    int r;
+    r = ERR_get_error();
+    fprintf(stderr, "openssl fips failed: %s\n", ERR_error_string(r, NULL));
```

Comment:
Oh, okay. I would have done

    fprintf(stderr, "openssl fips failed: %s\n",
        ERR_error_string(ERR_get_error(), NULL));

Can we use that?